examples/README.md

```diff
@@ -1,49 +1,6 @@
-# Flax Examples
- 
-## Official examples maintained by the Flax core team
- 
-This folder contains a collection of examples of implementations of various architectures and training procedures.
```

Comment:
Hi @andsteing -- I noticed that much of the content from this README hasn't been ported to our new documentation. Specifically:
1. The "example matrix" diagram
2. The description of goals for our examples
3. Links and suggestions from the broader ecosystem (we may want to reach out to the package authors linked to from here so that all the packages are ported to Linen).

Was there an intention to add `linen_examples/README.md`, or to add back some of this content directly into RTD? Or something else? Happy to help, just wanted to see if there had been prior thinking about this.

Reply:
I think I will first update and move to `linen_examples/README.md`. Not sure if we want to have it in RTD or keep it as a README. Let's discuss this and move it in a second step if we decide for RTD.

Reply:
Removed in #669